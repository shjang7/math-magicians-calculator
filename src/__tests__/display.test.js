import React from 'react';
import { render } from '@testing-library/react';
import Display from '../components/display';

describe('<Display />', () => {
  it('matches snapshot', () => {
    const { container } = render(<Display result="100" />);
    expect(container).toMatchSnapshot();
  });

  it('shows the props correctly', () => {
    const { getByText } = render(<Display result="100" />);
    getByText('100');
  });
});
