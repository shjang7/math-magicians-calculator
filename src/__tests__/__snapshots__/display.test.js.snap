// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display /> matches snapshot 1`] = `
<div>
  <div
    class="display d-flex"
  >
    100
  </div>
</div>
`;
